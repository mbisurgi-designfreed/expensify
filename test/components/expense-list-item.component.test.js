import React from 'react';
import { shallow } from 'enzyme';

import ExpenseListItem from '../../src/components/expense-list-item.component';
import expenses from '../fixtures/expenses';

test('should render ExpenseListItem correctly with expense', () => {
    const expense = expenses[0];
    const wrapper = shallow(<ExpenseListItem key={expense.id} {...expense} />);

    expect(wrapper).toMatchSnapshot();
});