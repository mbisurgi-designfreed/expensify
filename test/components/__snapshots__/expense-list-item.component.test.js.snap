// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly with expense 1`] = `
<div>
  <h3>
    Gum
  </h3>
  <p>
    $1.95
    -
    December 31st, 1969
  </p>
  <Link
    replace={false}
    to="/edit/1"
  >
    Edit
  </Link>
</div>
`;
